From: Carlos Maddela <e7appew@gmail.com>
Date: Thu, 12 Jan 2017 10:54:39 +1100
Subject: Fix spelling errors.

Description: Fix spelling errors.
Author: Carlos Maddela <e7appew@gmail.com>
Origin: vendor
Forwarded: https://github.com/sahib/rmlint/pull/203
Applied-Upstream: https://github.com/sahib/rmlint/commit/6742160eb42909a679506dfc964c77af52a58bca
Last-Update: 2017-01-12
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
---
 docs/rmlint.1.rst | 2 +-
 lib/shredder.c    | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/docs/rmlint.1.rst b/docs/rmlint.1.rst
index 10f67d0..db6fc68 100644
--- a/docs/rmlint.1.rst
+++ b/docs/rmlint.1.rst
@@ -332,7 +332,7 @@ Original Detection Options
 
     Sort the files in a group of duplicates into originals and duplicates by
     one or more criteria. Each criteria is defined by a single letter (except
-    **r** and **x**). Mutliple criterias may be given as string, where the
+    **r** and **x**). Multiple criteria may be given as string, where the
     first criteria is the most important. If one criteria cannot decide between
     original and duplicate the next one is tried.
 
diff --git a/lib/shredder.c b/lib/shredder.c
index 631cb16..afac3da 100644
--- a/lib/shredder.c
+++ b/lib/shredder.c
@@ -1364,7 +1364,7 @@ static bool rm_shred_reassign_checksum(RmShredTag *main, RmFile *file) {
             rm_log_debug_line("%s=%s was read from cache.", hexstring, file_path);
         } else {
             rm_log_warning_line(
-                "Unable to read external checksum from interal cache for %s", file_path);
+                "Unable to read external checksum from internal cache for %s", file_path);
             file->has_ext_cksum = 0;
             group->has_only_ext_cksums = 0;
         }
